client/src/cordova/plugin/apple/src/OutlinePlugin.swift

```diff
@@ -154,10 +154,12 @@ class OutlinePlugin: CDVPlugin {
     DDLogInfo("Invoking Method \(methodName) with input \(input)")
     Task {
       guard let result = OutlineInvokeMethod(methodName, input) else {
+        DDLogInfo("InvokeMethod \(methodName) got nil result")
         return self.sendError("unexpected invoke error", callbackId: command.callbackId)
       }
       if result.error != nil {
         let errorJson = marshalErrorJson(error: OutlineError.platformError(result.error!))
+        DDLogInfo("InvokeMethod \(methodName) failed with error \(errorJson)")
```

Comment:
Warning ?  Considering it might be user errors.
```suggestion
        DDLogWarn("InvokeMethod \(methodName) failed with error \(errorJson)")
```

Reply:
I disagree with this one. The invoked method may fail for various reasons (e.g. connectivity) and it's not necessarily something to report in the logs here. It's up to the caller to decide.
I had this to debug the calls during development.